pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -83,6 +90,7 @@
     "@types/sinon": "^9.0.4",
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
+    "@typescript-eslint/utils": "5.26.0",
```

Comment:
Looks like this is a net-new dependency! We can probably remove this line.

Reply:
`eslint` fails with the following error in the absence of this new dependency:

```
Error: Failed to load plugin '@cockroachlabs/eslint-plugin-crdb' declared in '--config': Cannot find module '@typescript-eslint/utils'
Require stack:
- /private/var/tmp/_bazel_ricky/be70b24e7357091e16c49d70921b7985/sandbox/darwin-sandbox/6184/execroot/com_github_cockroachdb_cockroach/bazel-out/darwin_arm64-fastbuild/bin/pkg/ui/workspaces/cluster-ui/eslint.sh.runfiles/com_github_cockroachdb_cockroach/pkg/ui/workspaces/cluster-ui/node_modules/.aspect_rules_js/@cockroachlabs+eslint-plugin-crdb@0.0.0/node_modules/@cockroachlabs/eslint-plugin-crdb/dist/require-antd-style-import.js
- /private/var/tmp/_bazel_ricky/be70b24e7357091e16c49d70921b7985/sandbox/darwin-sandbox/6184/execroot/com_github_cockroachdb_cockroach/bazel-out/darwin_arm64-fastbuild/bin/pkg/ui/workspaces/cluster-ui/eslint.sh.runfiles/com_github_cockroachdb_cockroach/pkg/ui/workspaces/cluster-ui/node_modules/.aspect_rules_js/@cockroachlabs+eslint-plugin-crdb@0.0.0/node_modules/@cockroachlabs/eslint-plugin-crdb/dist/index.js

```